packages/ui-app/src/Params/Param/Balance.tsx

```diff
@@ -0,0 +1,192 @@
+// Copyright 2017-2018 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { ApiProps } from '@polkadot/ui-react-rx/types';
+import { I18nProps } from '../../types';
+import { Props as BareProps } from '../types';
+
+import BN from 'bn.js';
+import React from 'react';
+import withApi from '@polkadot/ui-react-rx/with/api';
+
+import isValidBalance from '../../util/isValidBalance';
+import Bare from './Bare';
```

Comment:
Move this import to be last, i.e. first imports external to us, then our import.